frontend/public/components/utils/volume-kebab.tsx

```diff
@@ -0,0 +1,43 @@
+import * as _ from 'lodash-es';
+import * as React from 'react';
+import { asAccessReview, Kebab, KebabOption } from './index';
+import { K8sKind, K8sResourceKind, K8sResourceKindReference } from '../../module/k8s';
+import { connectToModel } from '../../kinds';
+import { RowVolumeData } from '../volumes-table';
+import { removeVolumeModal } from '../modals';
+
+export const VolumeKebabMenu: VolumeKebabFactory = {
```

Comment:
Since this is only used by `VolumeTable`, I would probably just move this into the same file.